src/openms/source/FORMAT/FileHandler.cpp

```diff
@@ -851,6 +827,22 @@ if (first_line.hasSubstring("File	First Scan	Last Scan	Num of Scans	Charge	Monoi
     }
     break;
 
+    case FileTypes::MGF:
```

Comment:
lovely switch case blocks ;)
Someone gets around doing a refactoring seminar at some point this class would be a great choice :P

Reply:
As long as I get some time and therapy to recover from the refactoring.